docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Gatsby v2 beta launch!
+date: "2018-06-14"
+author: "Mike Allanson"
+tags: ["v2", "gatsby"]
+---
+
+We’re excited to announce that today we shipped the first beta for Gatsby v2! 35 contributors have made over 300 commits to v2 since Gatsby v1 was released in July 2017.
+
+## What's coming in v2?
+
+Gatsby v2 builds on the foundations of v1 to introduce a range of improvements:
+
+- Out With Layouts, In With StaticQuery
+
+  Layouts were a common source of confusion in v1. After an RFC and [informative community discussion](https://github.com/gatsbyjs/rfcs/blob/master/text/0002-remove-special-layout-components.md), we decided to remove them and create `StaticQuery` which allows GraphQL queries to be added to any component. For more information on migrating your layouts, check out our Life After Layouts post (coming soon).
+
+- More consistent APIs
```

Comment:
Maybe the header should be "Improve APIs" as only some of the changes were for consistency.